### ISSUE TITLE
[Bug]: "Not enough funds" 

### ISSUE THREAD

**Issue by SPEAlbertooo** [metaplex-foundation/metaplex#1977]
### Which package is this bug report for?

candy machine ui

### Issue description

"Not enough funds" even if it takes sol from wallet. Over and over. 

### Command

```shell
C:\Users\PC\Desktop\MintSite>npx ts-node ./metaplex/js/packages/cli/src/candy-machine-v2-cli.ts upload C:\Users\PC\Desktop\MintSite\assets --env mainnet-beta --keypair [keypair].json --config-path config.json
```

### Relevant log output

```shell
[0] out of [150] items have been uploaded
Starting upload for [100] items, format {"mediaExt":".png","index":"0"}
Saved bundle upload result to cache.
Computed Bundle range, including 100 file pair(s) totaling 0.532MB.
Processing file groups...
Progress: [████████████████████████████████████████] 100% | 100/100
Uploading bundle via Bundlr... in multiple transactions
0.000214478 SOL to upload 0.668MB with buffer. Sending fund txn...
Successfully funded Arweave Bundler, starting upload
Progress: [███████████████████░░░░░░░░░░░░░░░░░░░░░] 46% | 140/300
Could not complete Bundlr tx upload successfully, exiting due to:  Error: Not enough funds to send data
    at NodeUploader.dataItemUploader (C:\Users\PC\Desktop\MintSite\metaplex\js\node_modules\@bundlr-network\client\build\common\upload.js:43:23)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async uploadTransaction (C:\Users\PC\Desktop\MintSite\metaplex\js\packages\cli\src\helpers\upload\arweave-bundle.ts:642:15)
    at async C:\Users\PC\Desktop\MintSite\metaplex\js\packages\cli\src\helpers\upload\arweave-bundle.ts:652:17
    at async uploadTransaction (C:\Users\PC\Desktop\MintSite\metaplex\js\packages\cli\src\helpers\upload\arweave-bundle.ts:642:15)
    at async C:\Users\PC\Desktop\MintSite\metaplex\js\packages\cli\src\helpers\upload\arweave-bundle.ts:652:17
    at async uploadTransaction (C:\Users\PC\Desktop\MintSite\metaplex\js\packages\cli\src\helpers\upload\arweave-bundle.ts:642:15)
    at async PromisePoolExecutor.handler (C:\Users\PC\Desktop\MintSite\metaplex\js\packages\cli\src\helpers\upload\arweave-bundle.ts:656:13)
```

### Operating system

Windows

### Priority this issue should have

High

### Check the Docs First

- [X] I have checked the docs and it didn't solve my issue

**Comment by stegaBOB** [comment]
Pretty sure you aren't on the latest version. Try pulling from master and reupload. This should be fixed.

**Comment by jjclinton** [comment]
> Pretty sure you aren't on the latest version. Try pulling from master and reupload. This should be fixed.

I did git clone https://github.com/metaplex-foundation/metaplex.git 2 days ago. That means I'm on master right? I doubt if this is the issue, because I tried installing multiple times, switching from Windows to Linux as well and trying again. Keep getting this error when trying to mint with a wallet other than the wallet I created the Candy Machine with. I am on devnet by the way, and use Quicknode devnet RPC.
